Tests/RxSwiftTests/DisposeBagTest.swift

```diff
@@ -26,14 +26,14 @@ extension DisposeBagTest {
         let disposable1 = TestDisposable()
         let disposable2 = TestDisposable()
 
-        var disposeBag: DisposeBag? = DisposeBag()
+        var disposeBag: DisposeBag = DisposeBag()
```

Comment:
In Xcode 15, noncopyable types can't be used as generic arguments, so no optional for us. If Xcode 15 compatibility isn't a concern, this and similar changes can be rolled back.